### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by merveenoyan**
Fixing gradio demos, broken refs, images, added code examples and more formatting nits

### CODE REVIEW

File: chapters/en/Unit 5 - Generative Models/GANs & VAEs/StyleGAN.mdx [hunk 1]
```diff
@@ -51,26 +52,24 @@ In traditional GAN, the generator has to learn stochastic features on its own. B
 
 Instead, in StyleGAN, the authors hypothesize that by adding a noise map to the feature map in each block of the synthesis network (also known as the generator), each layer makes use of this information to produce diverse stochastic nature without trying to do all by its own like in traditional GANs. This turned out well.
 
-![Example for noise](https://huggingface.co/datasets/hwaseem04/Documentation-files/blob/main/CV-Course/noise.png)
+![Example for noise](https://huggingface.co/datasets/hwaseem04/Documentation-files/resolve/main/CV-Course/noise.png)
 
 ## Drawbacks of StyleGAN1 and the need for StyleGAN2;
```
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/GANs & VAEs/StyleGAN.mdx:57]:
```suggestion
## Drawbacks of StyleGAN1 and the need for StyleGAN2
```

File: chapters/en/Unit 5 - Generative Models/GANs & VAEs/StyleGAN.mdx [hunk 2]
```diff
@@ -81,28 +80,31 @@ There are also other changes introduced in StyleGAN2, but the above two are impo
 ## Drawbacks of StyleGAN2 and the need for StyleGAN3;
 The same set of authors of StyleGAN2 figured out the dependence of the synthesis network on absolute pixel coordinates in an unhealthy manner. This leads to the phenomenon called the aliasing effect.
 
-![Animation of aliasing](https://huggingface.co/datasets/hwaseem04/Documentation-files/blob/main/CV-Course/MP4%20to%20GIF%20conversion.gif)
+![Animation of aliasing](https://huggingface.co/datasets/hwaseem04/Documentation-files/resolve/main/CV-Course/MP4%20to%20GIF%20conversion.gif)
 Above, the animation is generated by interpolating the latent code w. You can clearly see that in the left image the texture pixels kind of fix to the location and only the high-level attribute (face pose/expression) changes. This exposes the artificiality when generating such animations. StyleGAN3 tackles this problem ground up and you can see the results from the animation on the right side.
 
 
 ## Use cases
```
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/GANs & VAEs/StyleGAN.mdx:87]:
```suggestion
## Use Cases
```

File: chapters/en/Unit 6 - Basic CV Tasks/segmentation.mdx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# What is Image Segmentation? 
+# Image Segmentation?
```
Comment (merveenoyan) [inline chapters/en/Unit 6 - Basic CV Tasks/segmentation.mdx:1]:
```suggestion
# Image Segmentation
```

**Review by merveenoyan — COMMENTED** [review]
Consistency nits

**Review by bellabf — APPROVED** [review]
LGTM :smile:

**Comment by merveenoyan** [comment]
I'm fixing latex atm, not to be merged yet

**Comment by merveenoyan** [comment]
@MKhalusova can you approve?

File: README.md [hunk 4]
```diff
@@ -139,7 +139,8 @@ If you want to refer to a section inside the text, you can do it like below 👇
 You can write codeblocks by wrapping it with three backticks. Please add the associated language code, e.g. `py` or `bash` after top backticks to enable language specific rendering of code blocks. 
 
 **LaTeX**
-You can write LaTeX by writing it like this: `$$...$$``
+You can write in-line LaTeX by writing it like this: ` \\( X )\\`
+You can write stand alone LaTeX by enclosing with  $$.
```
Comment (MKhalusova) [inline README.md:143]:
```suggestion
You can write stand alone LaTeX by enclosing with  `$$`.
```

File: chapters/en/Unit 0 - Welcome/welcome.mdx [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 Dear learner,
 
-Welcome to the community-driven course on computer vision. Computer vision is revolutionizing our world in many ways, from unlocking phones with facial recognition to analyzing medical images for disease detection, enhancing public safety through surveillance systems, monitoring wildlife, and creating new images. Together, we'll dive into the fascinating world of computer vision!
+Welcome to the **community-driven course on computer vision**. Computer vision is revolutionizing our world in many ways, from unlocking phones with facial recognition to analyzing medical images for disease detection, enhancing public safety through surveillance systems, monitoring wildlife, and creating new images. Together, we'll dive into the fascinating world of computer vision!
```
Comment (MKhalusova) [inline chapters/en/Unit 0 - Welcome/welcome.mdx:5]: I would suggest not mentioning "surveillance systems" as this application is in an ethical grey area.  
There are many other examples that we could use instead, e.g. autonomous vehicles, quality control (e.g. defects detection in manufacturing), agriculture applications (precise pest control, or plant health monitoring), disaster relief, image search, etc.

File: chapters/en/Unit 0 - Welcome/welcome.mdx [hunk 6]
```diff
@@ -98,7 +98,8 @@ The course is organized into multiple units, covering the fundamentals and delvi
 
 ## Meet our team
 
-This is made by the Hugging Face Community with love! Our goal was to create a computer vision course that is beginner-friendly and that could act as a resource for others. Around 60+ people from all over the world joined forces to make this project happen. Here we give them credit:
+This is made by the Hugging Face Community with love 💜! You can also join us in [here](https://github.com/johko/computer-vision-course).
```
Comment (MKhalusova) [inline chapters/en/Unit 0 - Welcome/welcome.mdx:101]:
```suggestion
This course is made by the Hugging Face Community with love 💜! Join us by adding your contribution [on GitHub](https://github.com/johko/computer-vision-course).
```

File: chapters/en/Unit 1 - Fundamentals/feature-extraction/feature_description.mdx [hunk 7]
```diff
@@ -1,5 +1,7 @@
 ﻿# Feature Description
 
+Features are attributes of the instances learnt by the model to later use to recognize new instances.
```
Comment (MKhalusova) [inline chapters/en/Unit 1 - Fundamentals/feature-extraction/feature_description.mdx:3]:
```suggestion
Features are attributes of the instances learnt by the model to be later used to recognize new instances.
```

File: chapters/en/Unit 3 - Vision Transformers/oneformer.mdx [hunk 8]
```diff
@@ -34,15 +40,17 @@ Lastly, let's talk about "Query-Text Contrastive Loss." Think of it as a way for
...
 
 In summary, OneFormer represents a significant step towards universal and accessible image segmentation. By introducing a single model capable of handling diverse segmentation tasks, the framework streamlines the segmentation process and reduces resource requirements.
 
 ## Example use of model
+
 ```python
```
Comment (MKhalusova) [inline chapters/en/Unit 3 - Vision Transformers/oneformer.mdx:54]: Looks like this should be two separate cells. A bash cell with pip install, and then a python cell with the rest of the code.

File: chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx [hunk 9]
```diff
@@ -1,10 +1,10 @@
-# Extras
+# Control over Diffusion Models
 
-## Personalisation
+## Dreambooth
 
-Although diffusion models and GANs can create many new, unique items, they can't always create the specific thing you need. Hence, you have to fine-tune a model, which usually requires a lot of data and computation. However, some techniques can be used to personalize a model with just a few examples.
+Although diffusion models and GANs can generate many unique images, they can't always generate the what you exactly need. Hence, you have to fine-tune a model, which usually requires a lot of data and computation. However, some techniques can be used to personalize a model with just a few examples.
```
Comment (MKhalusova) [inline chapters/en/Unit 5 - Generative Models/Diffusion models/Simple explanation and showcase of external resources.mdx:5]:
```suggestion
Although diffusion models and GANs can generate many unique images, they can't always generate what you need exactly. Hence, you have to fine-tune a model, which usually requires a lot of data and computation. However, some techniques can be used to personalize a model with just a few examples.
```

**Review by MKhalusova — APPROVED** [review]
Thanks for all the fixes! I left a couple of minor nits but otherwise looks good to me.

**Review by lunarflu — APPROVED** [review]
LGTM, thanks @merveenoyan ! ❤️ 🤗

**Pull request merged**